examples/77_blackwell_fmha/collective/fmha_fusion.hpp

```diff
@@ -225,8 +225,8 @@ struct CausalMask : NoMask {
     if constexpr (IsQBegin) {
       return std::min(trip_count, int(ceil_div(size<0>(tile_shape), size<1>(tile_shape))));
     } else {
-      const int offset_tile_q = get<1>(problem_size) % get<1>(tile_shape);
-      return std::min(trip_count, int(ceil_div(get<0>(tile_shape) + offset_tile_q, get<1>(tile_shape))));
+      const int corner_count = int((get<1>(problem_size) % get<1>(tile_shape) || get<0>(problem_size) % get<0>(tile_shape))) ; 
+      return std::min(trip_count, int(ceil_div(get<0>(tile_shape), get<1>(tile_shape))) + corner_count);
```

Comment:
This doesn't look correct, suppose tile_shape0 = 2, tile_shape1 = 4, and  `get<1>(problem_size) % get<1>(tile_shape) = 2 and  `get<0>(problem_size) % get<0>(tile_shape) == 0` (so previous calculation should be correct, because in the 0th dimension problem is evenly divisible. Previously we would have `ceil_div(2+2, 4) = 1`, now we are getting `corner_count=1, ceil_div(2, 4) + corner_count = 1 + 1 = 2`, which differs from previous behavior, and it shouldn't given that in dim 0 proclem_size is divisible by tile_shape